XCTestBootstrap/TestManager/FBTestDaemonConnection.m

```diff
@@ -186,10 +186,10 @@ - (DTXRemoteInvocationReceipt *)setupDaemonConnectionViaLegacyProtocol
   [receipt handleCompletion:^(NSNumber *version, NSError *error) {
     if (error) {
       [self.logger logFormat:@"Error in whitelisting response from testmanagerd: %@ (%@), ignoring for now.", error.localizedDescription, error.localizedRecoverySuggestion];
-      return;
+    } else {
+      self.daemonProtocolVersion = version.integerValue;
+      [self.logger logFormat:@"Got legacy whitelisting response, daemon protocol version is 14"];
     }
-    self.daemonProtocolVersion = version.integerValue;
-    [self.logger logFormat:@"Got legacy whitelisting response, daemon protocol version is 14"];
     self.state = FBTestDaemonConnectionStateReadyToExecuteTestPlan;
```

Comment:
Yes, that makes sense. It's a bit strange that Xcode itself is also just trying the different `XPC` messages and is ignoring the errors coming back. We could be a bit smarter than that and check what Simulator version is being used and depending on that we select the correct `_IDE...` methods to invoke, instead of try-and-error :).